### DISCUSSION TITLE
Papers that cite Herbie

### DISCUSSION THREAD

**Discussion by blaylockbk** [blaylockbk/Herbie#329] [Show and tell]
This is a collection of papers that cite Herbie in peer-reviewed literature. If you see Herbie used in a paper, please tell everyone about it here!

**Comment by blaylockbk** [comment]
Santellanes and Melgar 2024

> Santellanes, S. R. and Melgar, D.: Contemporary measurements of the background open ocean tsunami spectrum using the Deep-ocean Assessment and Reporting of Tsunami (DART) stations, EGUsphere [preprint], https://doi.org/10.5194/egusphere-2024-1614, 2024.

**Comment by blaylockbk** [comment]
Baldwin et al. 2023
> Baldwin, M. E., H. D. Reeves, and A. A. Rosenow, 2023: Evaluation of a Probabilistic Subfreezing Road Temperature Nowcast System Based on Machine Learning. Wea. Forecasting, 38, 2427–2440, https://doi.org/10.1175/WAF-D-23-0137.1.

**Comment by blaylockbk** [comment]
Gaudet et al. 2024
> Gaudet, L. C., K. J. Sulia, R. D. Torn, and N. P. Bassill, 2024: Verification of the Global Forecast System, North American Mesoscale Forecast System, and High-Resolution Rapid Refresh Model Near-Surface Forecasts by Use of the New York State Mesonet. Wea. Forecasting, 39, 369–386, https://doi.org/10.1175/WAF-D-23-0094.1.

**Comment by blaylockbk** [comment]
Hennon et al. 2022
> Hennon, C. C., A. Coleman, and A. Hill, 2022: Short-Term Weather Forecast Skill of Artificial Neural Networks. Wea. Forecasting, 37, 1941–1951, https://doi.org/10.1175/WAF-D-22-0009.1.

**Comment by williamhobbs** [comment]
Millstein et al. 2023
> Millstein, D., S. Jeong, A. Ancell, R. Wiser, 2023: A database of hourly wind speed and modeled generation for US wind plants based on three meteorological models. Sci Data 10, 883. https://doi.org/10.1038/s41597-023-02804-w.

**Comment by williamhobbs** [comment]
Two that are NOT PEER REVIEWED, but maybe still worth listing:

EPRI (Larson et al.) 2023
> Guidance on Weather Datasets for Studies: Data Sources and Use Cases for Historical Datasets. 
EPRI, Palo Alto, CA: 2023. 3002024375. Online: https://www.epri.com/research/products/000000003002024375

EPRI (Larson et al.) 2024
> Guidance on Numerical Weather Prediction: Models and Data Sources. EPRI, Palo Alto, 
CA: 2024. 3002027745. Online: https://www.epri.com/research/products/000000003002027745

**Comment by blaylockbk** [comment]
Manshausen et al. (2024)

> Manshausen, P., and Coauthors, 2024: Generative data assimilation of sparse weather station observations at kilometer scales. https://arxiv.org/abs/2406.16947.

**Comment by blaylockbk** [comment]
Ghimire et al. 2021

> Ghimire, G. R., W. F. Krajewski, and F. Quintero, 2021: Scale-Dependent Value of QPF for Real-Time Streamflow Forecasting. J. Hydrometeor., 22, 1931–1947, https://doi.org/10.1175/JHM-D-20-0297.1.

**Comment by williamhobbs** [comment]
Hobbs 2024 (conference proceeding, not peer reviewed)
> Hobbs, W. B., 2024. Making Regional Probabilistic Solar Forecasts from Deterministic Forecasts with Open-Source Tools and Data. 2024 IEEE 52nd Photovoltaic Specialist Conference (PVSC). https://doi.org/10.1109/PVSC57443.2024.10748736 (preprint: https://doi.org/10.36227/techrxiv.172114760.07962823/v1)

**Comment by williamhobbs** [comment]
Bai, Mingliang et al. 2025

> Bai, Mingliang, Zuliang Fang, Shengyu Tao, Siqi Xiang, Jiang Bian, Yanfei Xiang, Pengcheng Zhao et al. "Ultrafast 24-h solar irradiance forecasts outperform numerical weather predictions across the USA." Cell Reports Physical Science (2025). https://doi.org/10.1016/j.xcrp.2025.102996

(this paper also used goes2go...)

**Comment by williamhobbs** [comment]
Hobbs et al. 2026

> W. B. Hobbs and D. Joshi, "Using Open-Source Forecasts for Solar Plant Maintenance Outage Scheduling Can Reduce Lost Energy," in IEEE Journal of Photovoltaics, doi: [10.1109/JPHOTOV.2025.3646334](https://doi.org/10.1109/JPHOTOV.2025.3646334)

**Comment by williamhobbs** [comment]
Shumacher et al. 2026

> Russ S. Schumacher, R. S. and Hill, A. J., 2026: Extreme Precipitation in the Contiguous United States in Gridded Analyses and a Convection-Permitting Model Simulation, Journal of Hydrometeorology, https://doi.org/10.1175/JHM-D-25-0212.1
